Packs/ApiModules/Scripts/TAXII2ApiModule/TAXII2ApiModule.py

```diff
@@ -700,9 +740,14 @@ def parse_tool(self, tool_obj: Dict[str, Any]) -> List[Dict[str, Any]]:
             "killchainphases": kill_chain_phases,
             "tool_types": tool_obj.get("tool_types", []),
             "aliases": tool_obj.get('aliases', []),
-            "tool_version": tool_obj.get('tool_version', ''),
-            "tags": list(set(self.tags))
+            "tool_version": tool_obj.get('tool_version', '')
```

Comment:
Why have you kept the handling of tool_version before custom fields and the handling of `tags` was changed after the addition of custom fields? (I saw this in other IOCs as well, so asking in general)

Reply:
In other fields, we overwrite the data if it exists in custom fields,
But in tags, we want to add them and return all to the list